tendrl/gluster_integration/objects/gluster_brick_path/__init__.py

```diff
@@ -0,0 +1,23 @@
+from tendrl.commons import objects
+
+
+class GlusterBrickDir(objects.BaseObject):
+    def __init__(
+        self,
+        default_brick_dir=None,
+        *args,
+        **kwargs
+    ):
+        super(GlusterBrickDir, self).__init__(*args, **kwargs)
+
+        self.default_brick_dir = default_brick_dir if default_brick_dir else \
```

Comment:
Hey, why do you need object to encapsulate a config item?

This default_brick_dir can be accessed directly anywhere via NS.config.data.get('gluster_bricks_dir', "/tendrl_gluster_bricks")